libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1572,6 +1572,9 @@ void ExpressionCompiler::appendExternalFunctionCall(
 	bool returnSuccessCondition = funKind == FunctionType::Kind::BareCall || funKind == FunctionType::Kind::BareCallCode || funKind == FunctionType::Kind::BareDelegateCall;
 	bool isCallCode = funKind == FunctionType::Kind::BareCallCode || funKind == FunctionType::Kind::CallCode;
 	bool isDelegateCall = funKind == FunctionType::Kind::BareDelegateCall || funKind == FunctionType::Kind::DelegateCall;
+	bool useStaticCall =
```

Comment:
I think `use` suits this better, but the other two are named `is` so perhaps just go with `isStaticCall` or alternatively rename the other two to `use`?

Reply:
It's a different thing. I would call it `isStaticCall` out of the function type, but `useStaticCall` tells whether the compiler is also allowed to use the opcode due to pragmas.